src/style.css

```
@@ -0,0 +1,120 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
```

```suggestion
.search-form {
  display: flex;
  justify-content: center;
  margin: 20px auto 0;
  width: 100%;
  height: 70px;
}
```
as example what i meant in comment about redundant modifier classes